packages/@tinacms/toolkit/src/packages/fields/plugins/BlocksFieldPlugin/BlockSelectorBig.tsx

```diff
@@ -167,6 +167,7 @@ export const BlockSelectorBig = ({
                         <CardColumns className="pt-3">
                           {uncategorized.map(([name, template]) => (
                             <BlockCard
+                              key={`${template}-${name}`}
```

Comment:
Noticed I was getting a nasty `missing key` React Warning when using the `tina-cloud-starter`; this should fix that.

Reply:
👍